lib/const.rb

```diff
@@ -72,15 +72,20 @@
   CREW_BUILD = 'x86_64-cros-linux-gnu'
 end
 
+CREW_COMMON_FLAGS = "'-Os -pipe -fno-lto -fuse-ld=gold'"
+CREW_COMMON_FNO_LTO_FLAGS = "'-Os -pipe -fno-lto -fuse-ld=gold'"
```

Comment:
Isn't this a duplicate of `CREW_COMMON_FLAGS`?

Reply:
Oh thank you, one of those is supposed to say `-flto=auto`